app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/dashboard.tsx

```diff
@@ -1097,334 +1097,357 @@ function Dashboard(props: DashboardProps) {
+                                        </span>
                                     }
-                                    const doOpenAsFolder = () => {
-                                        // FIXME[sb]: Uncomment once backend support
-                                        // is in place.
-                                        // The following code does not typecheck
-                                        // since `ProjectId`s are not `DirectoryId`s.
-                                        // enterDirectory(projectAsset)
-                                    }
-                                    // This is not a React component even though it contains JSX.
-                                    // eslint-disable-next-line no-restricted-syntax
-                                    const doRename = () => {
+                                    columns={columnsFor(columnDisplayMode, backend.type).map(
+                                        column => ({
+                                            id: column,
+                                            heading: ColumnHeading(
+                                                column,
+                                                backendModule.AssetType.project
+                                            ),
+                                            render: renderer(
+                                                column,
+                                                backendModule.AssetType.project
+                                            ),
+                                        })
+                                    )}
+                                    onClick={(projectAsset, event) => {
+                                        event.stopPropagation()
+                                        unsetModal()
+                                        setSelectedAssets(
+                                            event.shiftKey
+                                                ? [...selectedAssets, projectAsset]
+                                                : [projectAsset]
+                                        )
+                                    }}
+                                    onContextMenu={(projectAsset, event) => {
+                                        event.preventDefault()
+                                        event.stopPropagation()
+                                        const doOpenForEditing = () => {
+                                            unsetModal()
+                                            setProjectEvent({
+                                                type: projectActionButton.ProjectEventType.open,
+                                                projectId: projectAsset.id,
+                                            })
+                                        }
+                                        const doOpenAsFolder = () => {
+                                            // FIXME[sb]: Uncomment once backend support
```

Comment:
FIXMEs should be fixed before a PR is merged.
TODOs can be left in if they're being addressed in a later PR.
TODOs must be associated with an issue in the issue tracker so we can keep track of them.

Seems like this should be changed to a TODO and associated with an issue. This applies to the other FIXMEs/TODOs touched by this PR.